test/components/prometheus-operator/prometheus_operator_test.go

```diff
@@ -67,3 +78,88 @@ func TestPrometheusOperatorDeployment(t *testing.T) {
 
 	testutil.WaitForDaemonSet(t, client, namespace, "prometheus-operator-prometheus-node-exporter", retryInterval, timeout)
 }
+
+// nolint:funlen
+func TestGrafanaLoadsEnvVars(t *testing.T) {
+	kubeconfig := testutil.KubeconfigPath(t)
+
+	t.Logf("using KUBECONFIG=%s", kubeconfig)
```

Comment:
We're not very consistent with capitalization on errors. I've created an issue so we can address this: https://github.com/kinvolk/lokomotive/issues/637

Reply:
I think this was to highlight it similar to env var.